packages/strapi-design-system/src/Select/Select.js

```diff
@@ -223,6 +228,7 @@ export const Select = ({
                 onMouseDown={handleMouseDown}
                 tabIndex={-1}
                 disabled={disabled}
+                title="Carret Down Button"
```

Comment:
What is this? In any case these need to be translateable ...

Reply:
I need it to fix the ay11 problems

Reply:
I understand that, but this string needs to be translateable. The DS should not assume a language and allow to pass in all localized strings through props.

Reply:
ok now I pass it as a prop